### ISSUE TITLE
Background Tracking Enabled Notification Bar

### ISSUE THREAD

**Issue by lkaragol** [christocracy/cordova-plugin-background-geolocation#125]
I'm using Cordova 4.2.0 and IOS 8.1. When I add this plugin to my project, even I don't call the bgGeo.start() function, background tracking enabled notification is shown independent of usage. I've tried to call start and stop functions but no difference. Notification is always visible when I push the home button.

By the way, I'm marking "Location Updates" in "Background Mode" part of Capabilities section of xcode project.

![screen shot 2015-01-10 at 12 10 41](https://cloud.githubusercontent.com/assets/7867743/5691163/ca4c9d92-98c1-11e4-98b1-549074ab8ec4.png)

I'd like to show this notification only during background tracking usage. Is this a known issue or am I doing somethings wrong?

**Comment by christocracy** [comment]
Those notifications are merely a part of the *debug* system -- not for
normal usage.

Look at your configure params and read the README

On Saturday, January 10, 2015, H.Levent KARAGÖL <notifications@github.com>
wrote:

> I'm using Cordova 4.2.0 and IOS 8.1. When I add this plugin to my project,
> even I don't call the bgGeo.start() function, background tracking enabled
> notification is shown independent of usage. I've tried to call start and
> stop functions but no difference. Notification is always visible when I
> push the home button.
>
> By the way, I'm marking "Location Updates" in "Background Mode" part of
> Capabilities section of xcode project.
>
> [image: screen shot 2015-01-10 at 12 10 41]
> <https://cloud.githubusercontent.com/assets/7867743/5691163/ca4c9d92-98c1-11e4-98b1-549074ab8ec4.png>
>
> I'd like to show this notification only during background tracking usage.
> Is this a known issue or am I doing somethings wrong?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/christocracy/cordova-plugin-background-geolocation/issues/125>
> .
>


-- 
Snet from Gmail Mobile

**Comment by lkaragol** [comment]
I've already read the README for many times, thanks. You can find my config settings below. As you see,  "debug" flag is assigned as "false". I don't think, this notification related with debug. This is a standard IOS notification to notify users about that their location is being tracked. My issue is that, even I call the "stop" function, blue notification bar doesn't disappear.

bgGeo.configure(callbackFn, failureFn, {
                url: 'https://www.zzz.com/',
                params: {},
                desiredAccuracy: 5,
                stationaryRadius: 10,
                distanceFilter: 10,
                notificationTitle: "foo",
                notificationText: "foo",
                activityType: "AutomotiveNavigation",
                debug: false,
                stopOnTerminate: false
            });

**Comment by christocracy** [comment]
Don't know.

On Saturday, January 10, 2015, H.Levent KARAGÖL <notifications@github.com>
wrote:

> I've already read the README for many times, thanks. You can find my
> config settings below. As you see, "debug" flag is assigned as "false". I
> don't think, this notification related with debug. This is a standard IOS
> notification to notify users about that their location is being tracked. My
> issue is that, even I call the "stop" function, blue notification bar
> doesn't disappear.
>
> bgGeo.configure(callbackFn, failureFn, {
> url: 'https://www.zzz.com/',
> params: {},
> desiredAccuracy: 5,
> stationaryRadius: 10,
> distanceFilter: 10,
> notificationTitle: "foo",
> notificationText: "foo",
> activityType: "AutomotiveNavigation",
> debug: false,
> stopOnTerminate: false
> });
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/christocracy/cordova-plugin-background-geolocation/issues/125#issuecomment-69459454>
> .
>


-- 
Snet from Gmail Mobile

**Comment by christocracy** [comment]
Try using the SampleApp in plugin.  Documented in REaDME.

Let me know if it occurs there.

On Saturday, January 10, 2015, H.Levent KARAGÖL <notifications@github.com>
wrote:

> I've already read the README for many times, thanks. You can find my
> config settings below. As you see, "debug" flag is assigned as "false". I
> don't think, this notification related with debug. This is a standard IOS
> notification to notify users about that their location is being tracked. My
> issue is that, even I call the "stop" function, blue notification bar
> doesn't disappear.
>
> bgGeo.configure(callbackFn, failureFn, {
> url: 'https://www.zzz.com/',
> params: {},
> desiredAccuracy: 5,
> stationaryRadius: 10,
> distanceFilter: 10,
> notificationTitle: "foo",
> notificationText: "foo",
> activityType: "AutomotiveNavigation",
> debug: false,
> stopOnTerminate: false
> });
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/christocracy/cordova-plugin-background-geolocation/issues/125#issuecomment-69459454>
> .
>


-- 
Snet from Gmail Mobile

**Comment by lkaragol** [comment]
Ok, I think I've solved. It may not related with the plugin, It may be an IOS issue. I realized that, If you select the "While Using the App" option from Settings > Privacy > Location Services > MyApp > Allow Location Access, blue notification bar always stay visible on home screen independent of usage. If you select "Always", It disappears.

I'm not sure why IOS thinks plugin using the location service even before call the start function but Always options can be an alternative solution for me.
